unity-client/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/GlobalUsersPositionMarker/Utils/UserPositionMarkerUtils.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+internal class ExclusionArea
+{
+    public Vector2Int position;
+    public int area;
+
+    public bool Contains(Vector2Int coords)
+    {
+        return (coords - position).sqrMagnitude <= area * area;
+    }
+}
+
+internal class ScenesFilter
```

Comment:
Could we add a documentation header here to explain what is the responsibility of this class? Thanks!